x-pack/test_serverless/api_integration/test_suites/common/alerting/summary_rules.ts

```diff
@@ -0,0 +1,361 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { padStart } from 'lodash';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { createIndexConnector, createEsQueryRule, runRule } from './helpers/alerting_api_helper';
+import {
+  createIndex,
+  getDocumentsInIndex,
+  waitForDocumentInIndex,
+} from './helpers/alerting_wait_for_helpers';
+
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esClient = getService('es');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+
+  describe('Alerting summary rules', () => {
+    const RULE_TYPE_ID = '.es-query';
+    const ALERT_ACTION_INDEX = 'alert-action-es-query';
+    let actionId: string;
+    let ruleId: string;
+
+    afterEach(async () => {
+      await supertest
+        .delete(`/api/actions/connector/${actionId}`)
+        .set('kbn-xsrf', 'foo')
+        .set('x-elastic-internal-origin', 'foo');
+      await supertest
+        .delete(`/api/alerting/rule/${ruleId}`)
+        .set('kbn-xsrf', 'foo')
+        .set('x-elastic-internal-origin', 'foo');
+      await esClient.deleteByQuery({
+        index: '.kibana-event-log-*',
+        query: { term: { 'kibana.alert.rule.consumer': 'alerts' } },
+      });
+      await esDeleteAllIndices([ALERT_ACTION_INDEX]);
+    });
+
+    it('should schedule actions for summary of alerts per rule run', async () => {
+      actionId = await createIndexConnector({
+        supertest,
+        name: 'Index Connector: Alerting API test',
+        indexName: ALERT_ACTION_INDEX,
+      });
+      expect(actionId).not.to.be(undefined);
```

Comment:
If it helps to remove repetition, you could make the helper functions ensure the request succeeded before returning from the helper, that way the helper function throws in the failure case and removes the need to check for the existence of the result in the test file. The error thrown there would also contain more information (message, status code, etc) which will make things easier when it comes to debugging the issue in a PR (otherwise we'd get a message saying undefiend not expected to be undefined).

Ex: Adding .expect(200) to the request chain.